databases/week2/more-queries.sql

```diff
@@ -0,0 +1,24 @@
+USE hyf;
+select * from task;
+
+-- 1 
+SELECT  user.name, user.email FROM user
+WHERE user.email REGEXP '@spotify.com$';
+
+-- 2
+SELECT * FROM task
+JOIN user 
+ON task.id = user.id
+JOIN status
+ON task.id = task.status_id
+WHERE user.name REGEXP 'Donald Duck' AND status.name = 'Not started' ;
+
+-- 3
```

Comment:
Same as above, you need to use the table user_task to connect the tables user and task, let me know if you need any help with this :)